autoscheduler/frontend/src/components/LandingPage/SelectTerm/SelectTerm.tsx

```diff
@@ -45,10 +45,17 @@ const SelectTerm: React.FC = () => {
     // Get the corresponding option given the term's description
     const term: string = termMap.get(option);
 
-    dispatch(setTerm(term));
-
-    // Redirect to the main page
-    navigate('/schedule');
+    // Redirect to the main page, term will be set and retrieved by API calls
+    // Ignore any errors (caused by cookies not being enabled) so that sessions aren't needed
+    // in order to set term
+    fetch(`sessions/set_last_term?term=${term}`, {
```

Comment:
How does this pass the session ID? Does fetch automatically send cookies?

Reply:
Yeah, session id is sent along with requests